extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -327,6 +331,36 @@ export class Bind {
     return this.history_;
   }
 
+  /**
+   * Scans the document for <amp-bind-macro> elements, and adds them to the bind-evaluator.
+   *
+   * Returns a promise that resolves after macros have been added.
+   *
+   * @return {!Promise<number>}
+   * @private
+   */
+  addMacros_() {
+    const elements =
+        childElementsByTag(this.ampdoc.getBody(), 'AMP-BIND-MACRO');
```

Comment:
True, we shouldn't restrict this to children of body if `amp-state` isn't. Let's just use `scopedQuerySelectorAll` then.